srcStatic/OP2Memory.cpp

```diff
@@ -6,8 +6,8 @@
 
 
 bool memoryPatchingEnabled = false;
-std::size_t loadOffset = 0;
-constexpr std::size_t ExpectedOutpost2Addr = 0x00400000;
+uintptr_t loadOffset = 0;
```

Comment:
What do you think about labeling all these std::uintptr_t?

I also noticed we have some uint64_t type definitions in OP2Utility. I was going to ask if they should be prepended to read std::uint64_t.

Reply:
Good question. All the examples I saw were without a `std::` prefix. That seems to stem from these coming from C99. The C language doesn't have namespaces, only C++ does.

Turns out the `std::` prefix seems to work.

I suppose I find it a bit of a nuisance when seemingly primitive data types must be namespace prefixed, though that is the suggestion for `std::size_t`. I may follow up with an update once I've convinced myself it really is standard, and proper to do.

Reply:
I had already tested the std namespace on the types; it works on MSVC as well. 

It doesn't bother me that we are not using it, besides that it seems like a double standard when we do it for size_t but not others.

Compiled and worked fine on my local machine.